usl_pipeline/cloud_functions/main.py

```diff
@@ -347,6 +347,10 @@ def _build_feature_matrix(
             logging.info(
                 "Start generating flood feature matrix for chunk %s", chunk_path
             )
+            metastore.StudyAreaSpatialChunk(
+                id_=chunk_name,
+                error=firestore.DELETE_FIELD,
```

Comment:
Why do we clear the error here?

Reply:
Because if we do retry after some error happened in previous attempt to build unscaled matrix, we want to start from clean state again. But I can avoid clearing the error here if you think it's better to keep it till we succeed with building unscaled matrix (we do clear the error at that moment too).

Reply:
I think it'd be good to keep the error around until we actually succeed in case the next invocation is unable to write something out to avoid losing data.